src/main/java/gov/loc/repository/bagit/conformance/MetadataChecker.java

```diff
@@ -16,38 +16,39 @@
 import gov.loc.repository.bagit.reader.MetadataReader;
 
 /**
- * Part of the BagIt conformance suite. 
```

Comment:
As far as I can tell, this file only contains formatting changes, is that correct?